src/administration/integrations/webhooks.md

```diff
@@ -1,17 +1,97 @@
 # Generic Webhook
 
-This hook will allow you to capture any push events on platform and POST
-a JSON file describing the activity to the url of your choice. You can
-use this to further automate your Platform.sh workflow. 
-
-For example the `routes` object keys will give you urls against which you can 
-launch automated user testing.
+This hook will allow you to capture any push events on platform and POST a JSON file describing the activity to the url of your choice. You can use this to further automate your Platform.sh workflow.
```

Comment:
* JSON file is technically incorrect. There is no file involved. Maybe JSON data is more appropriate?
* The future tense (this hook will allow) sounds like this feature is not implemented yet? I dont know, just consider it and if you find it confusing feel free to reword it.